file_paths.md

```diff
@@ -163,39 +163,107 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
     Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    C:\ProgramData\PuppetLabs *                                              n/a
 
     C:\ProgramData\PuppetLabs\client-tools    # default client tool settings
         puppet-access.conf *
         puppet-orchestrator.conf *
 
-
     C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
         environments *                        # :environmentpath
           production *
             environment.conf *
+            hieradata *
             manifests *
             modules *
-        hiera.yaml *                          # :hiera_config
-        hieradata *                           # n/a
-        modules *                             # user modulepath
+        hiera.yaml                            # :hiera_config
+        modules                               # user modulepath
+
+    C:\ProgramData\PuppetLabs\mcollective *
+        client.cfg *
+        facts.yaml *
+        server.cfg *
 
-    C:\ProgramData\PuppetLabs\facter *                                       same
+    C:\ProgramData\PuppetLabs\puppet          # :confdir
+        auth.conf *                           # :rest_authconfig
+        autosign.conf                         # :autosign
+        binder_config.yaml                    # :binder_config
+        csr_attributes.yaml                   # :csr_attributes
+        custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
+        device.conf                           # :deviceconfig
+        fileserver.conf                       # :fileserverconfig
+        puppet.conf *                         # :config
+        routes.yaml                           # :route_file
+        ssl                                   # :ssldir
+
+    C:\ProgramData\PuppetLabs\pxp-agent
+        modules *                             # stores configuration files for pxp-agent modules
+            pxp-module-puppet.conf *          # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+        pxp-agent.conf                        # pxp-agent configuration file
+
+    C:\Program Files\Puppet Labs\bin *
+        environment.bat *                     # setup LOAD_PATH
```

Comment:
Sadly that script does a bunch of things. It might be good to ticket moving that logic elsewhere so that we can remove it in the future?

* sets [env_windows_installdir](https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L8) fact. Could be a core fact instead, since many things rely on it, e.g. PE.
* builds up [PATH](https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L25). Hopefully all of that logic goes away in the layout?
* sets [RUBYLIB](https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L27-L31). I think that logic can go away because ruby code is in `vendor_ruby`. Also I don't know that we need to convert to forward slashes any more.
* might be another way to [enable rubygems support](https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L33-L35)?
* overrides [trusted ssl directories](https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L37-L39)